wasm-engines/bench_wasm_and_native.sh

```diff
@@ -57,9 +58,7 @@ do
   /root/sentinel-minify-tool/wasm-utils/target/debug/wasm-minify "${filename}" "$dest"
 done
 
-
-
-# benchmark standalone wasm files in all the engines
-
+echo "running benchmarks"
 cd /benchrunner
-python3.7 main.py --wasmdir="${WASM_MINIFIED_DIR}" --csvfile="${CSV_WASM_RESULTS}" |& tee wasm-engines-run1.log
+python3.8 main.py --wasmdir="${WASM_MINIFIED_DIR}" --csvfile="${CSV_WASM_RESULTS}" |& tee wasm-engines-run1.log
```

Comment:
Why upgrade to 3.8?

Reply:
The repo isn't available by default in `ubuntu:20.04` image and it doesn't seem to break anything.

Reply:
You mean python3.7 is not available in ubuntu:20.04 ?